README.md

```diff
@@ -320,6 +320,10 @@ First, you need to install:
 * [uv](https://docs.astral.sh/uv/)
 * Ruby 2.6+
 * [Xcop](https://github.com/yegor256/xcop)
+* [LaTeX](https://www.latex-project.org/get/)
```

Comment:
@DvrkRain why it's `or`? We must install `pdflatex` _and_ `latexmk`. I suggest listing them one after another.